### ISSUE TITLE
Downloading whole pdb - different coordinates

### ISSUE THREAD

**Issue by danpf** [rcsb/mmtf-python#34]
Hello,
I'm getting very slightly different coordinates when using the hadoop database...

```
from pyspark import SparkContext, SparkConf

conf = SparkConf().setAppName("local")
sc = SparkContext(conf=conf )
reader = sc.sequenceFile('folder_containing_extracted_hadoop/")
pose_3KMA = reader.lookup('3KMA')[0]
pose_3KMA = mmtf.api.default_api.ungzip_data(pose_3KMA)
pose_3KMA = msgpack.unpackb(pose_3KMA.read())

pose = mmtf.MMTFDecoder()
pose.decode_data(pose_3KMA)
print(pose.x_coord_list[0])
```
printed:
```
-28.8
```
but the real CA coordinate is
```
-28.761999
```

Am I missing a step?

Also a function containing something similar to the below code might be useful in this library, it's mostly something I just scraped together...

**Comment by pwrose** [comment]
The Hadoop sequence files come in two flavors: full and reduced.

Which of the two files did you use?

The "reduced" files have a 0.1 Angstrom precision, so you might be using
the reduced version.

BTW, its easier to work with the mmtf-pyspark library (
https://github.com/sbl-sdsc/mmtf-pyspark), rather than the low-level API in
mmtf-python.

-Peter

**Comment by danpf** [comment]
Why is the reduced hadoop 0.1 A precision?

I downloaded an mmtf that was reduced and got more than 0.1A precision
```
(py36) >>> ~/mmtf-c/traverse.exe 3KMA.mmtf 
ATOM 2 CA . MET A 1 . -28.761999 34.069000 59.801998 23.070000 1.000000 C 0 
ATOM 10 CA . PHE A 2 . -28.695000 37.460999 58.098999 24.770000 1.000000 C 0 
ATOM 21 CA . ASN A 3 . -27.653000 41.096001 58.188999 25.980000 1.000000 C 0 
```

I don't mind the low level api... I guess it doesn't make sense to have spark code in this repo though..

*Side note I just thought of,
I know this isn't the right repo, so I can make an issue there as well... but would you know if the mmtf-cpp repo deprecated in favor of mmtf-c? the c one seems to be updated more often...

**Comment by pwrose** [comment]
A description of the reduced format can be found here:
http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005575
https://mmtf.rcsb.org/download.html

The reduced precision format can be used in applications, such as the
visualization of large structures or structural alignment of C-alpha traces.

We'll look into the issue with the higher precision you've observed in the
reduced file format.

-Peter

**Comment by danpf** [comment]
Sorry, I was unclear/terse...

I was making a unit test, and I was comparing the reduced mmtf file vs the reduced hadoop format.  I saw that the reduced was supposed to be 0.1A, but I took the mmtf file as the 'correct' implementation.

As someone that uses CA traces often, I would much rather have exact CA coordinates(the observed mmtf file format), and round them up with post-processing if necessary.  I'm not sure how much that would impact your space requirements of the current hadoop implementation, just my thoughts!
